### PR TITLE
Bartender Machine/Shunting Fix

### PR DESCRIPTION
**Pull request by ghost**
Fixes the Bartender machine, it can now hold glasses, dispense Space Cola, and a few drinks were removed that weren't supposed to be there in the Booze-o-Mat. AI shunting is now fixed as well.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -159,7 +159,7 @@
 	if(isrobot(user))
 		return
 
-	if(!istype(B, /obj/item/weapon/reagent_containers/glass || /obj/item/weapon/reagent_containers/food))
```
Comment (ZomgPonies) [inline code/modules/reagents/Chemistry-Machinery.dm:162]: if(!istype(B, /obj/item/weapon/reagent_containers/glass) || !istype(B, /obj/item/weapon/reagent_containers/food))

If you leave it like you did then chemists won't be able to put in beakers anymore

**Comment by ZomgPonies** [comment]
Remove the .int from this pull request and correct the coding thing I showed. You can do that by updating the pull request with MORE commits :)

**Comment by ZomgPonies** [comment]
Fixed manually myself

**Comment by Iamgoofball** [comment]
you guys are using a super duper old version of the booze dispenser and shit

the one on bs12 can be hacked to get goldshlager and patron

**Pull request closed without merging**